drools/src/main/java/com/datatorrent/drools/rules/DroolListener/RulesFiredListener.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2017 DataTorrent, Inc. ALL Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.drools.rules.DroolListener;
```

Comment:
package name shouldn't start with capital letter.
Also can we name it as rules.listener or rules.listener.drools, just to make sure if we add new rules engine tomorrow we are more accommodating